firebase.json

```diff
@@ -1,5 +0,0 @@
-{
-  "react-native": {
-    "messaging_ios_auto_register_for_remote_messages": false
```

Comment:
@jeanregisser mentioned on my last PR that doing this would move the Native prompt to Allow Notifications to the initial app opening, but I found that removing this file had no impact. The user is still prompted on the Home screen.

Reply:
Great!
Thanks yes I see the cloud messaging init waits for the [`VISIT_HOME`](https://github.com/valora-inc/wallet/blob/2c1fae9ddf2d4a16dc25bc11334fca7747574b1c/src/firebase/firebase.ts#L167) action now.
All good, sorry to have misled you.